src/utils/getAdminSub.js

```diff
@@ -1,7 +1,8 @@
 const config = require('../config');
 const getAwsPoolId = require('../api.v2/helpers/cognito/getAwsPoolId');
+const { ACCOUNT_ID } = require('../api.v2/constants');
 
-const adminEmail = 'cellenics_admin@listserv.med.harvard.edu';
+const adminEmail = process.env.AWS_ACCOUNT_ID === ACCOUNT_ID.HMS ? 'cellenics_admin@listserv.med.harvard.edu' : 'admin@biomage.net';
```

Comment:
would it make sense to have this kind of logic in a single location (like `domainSpecificContent.js` for the ui)?